lib/src/text-input/TextInput.tsx

```diff
@@ -487,6 +488,7 @@ const DxcTextInput = React.forwardRef<RefType, TextInputPropsType>(
                   }}
                   backgroundType={backgroundType}
                   tabIndex={tabIndex}
+                  title="Decrement"
```

Comment:
Same as above. Maybe something like "Decrement number" would be more useful to the user.

Reply:
Change by "Decrement value".